dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Microsoft.Windows.ApplicationModel.Resources.idl

```diff
@@ -4,9 +4,9 @@
 namespace Microsoft.Windows.ApplicationModel.Resources
 {
     [contractversion(1.0)]
-    apicontract MrtContract{};
+    apicontract MrtCoreContract{};
 
-    [contract(MrtContract, 1.0)]
+    [contract(MrtCoreContract, 1.0)]
```

Comment:
can you change 1.0 to 1?

Reply:
is this resolved?

Reply:
Doh! It was, but I forgot to push the commit :-(

Fixed by https://github.com/microsoft/WindowsAppSDK/pull/2384